### PR TITLE
[DO NOT MERGE] Update metrics data ingestion limit

### PR DESCRIPTION
**Pull request by aszczepaniksumo**
## [DO NOT MERGE] Purpose of this pull request

This pull request (PR) updates documentation of limits for late data.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by aszczepaniksumo** [comment]
I'm unable to add *DO NOT MERGE* label. Please add it as this needs to be merged during rollout.

**Comment by JV0812** [comment]
Label added.

**Comment by jpipkin1** [comment]
Per [Slack message](https://sumologic.slack.com/archives/C029XU6DCAC/p1689857027451559?thread_ts=1689802961.647319&cid=C029XU6DCAC) from @aszczepaniksumo, this PR replaces PR #2800.

File: blog-service/2023-07-20-metrics.md
Comment (jpipkin1) [inline blog-service/2023-07-20-metrics.md:15]:
```suggestion
We are making some changes to Metrics ingest and storage. Metrics reported with a timestamp older than 24 hours ago, or newer than 24 hours in the future from the time they are reported, will be dropped. These changes are made to make our metrics backend system more performant, efficient, and reliable.

The update should not have any impact on your usage of Sumo Logic, unless you were specifically sending old metrics data to the platform. Check out more details [here](/docs/metrics/manage-metric-volume/metric-ingestion-and-storage).
```

File: docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md
Comment (jpipkin1) [inline docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md:11]:
```suggestion
Sumo Logic does not store metrics data points that are more than 24 hours in the past or that are more than 24 hours in the future.
```

File: docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md
Comment (jpipkin1) [inline docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md:14]:
```suggestion
Too old or too new data points will be accepted by Sumo Logic, but they will not be stored.
```

File: docs/send-data/hosted-collectors/http-source/logs-metrics/index.md
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/http-source/logs-metrics/index.md:52]:
```suggestion
* Sumo Logic enforces limits on the volume of metrics and associated metadata you ingest. For more information, see [Data Limits for Metrics](/docs/metrics/manage-metric-volume/data-limits-for-metrics).
:::
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
Looks goods, @aszczepaniksumo. I only have a few comments. Click the **Commit suggestion** buttons on the comments if you want to accept them. Once you're done making changes, I'll approve the PR.

**Review by jpipkin1 — APPROVED** [review]
Thanks for making the updates, @aszczepaniksumo. Approved.

File: docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md
Comment (angadrandhawa1) [inline docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md:13]:
```suggestion
```

File: docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md
Comment (angadrandhawa1) [inline docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md:14]:
```suggestion
```

File: docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md
Comment (angadrandhawa1) [reply docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md:13]: We don't need to have the note since it is already covered above.

File: docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md
Comment (angadrandhawa1) [inline docs/metrics/manage-metric-volume/metric-ingestion-and-storage.md:15]:
```suggestion

```

**Review by angadrandhawa1 — APPROVED** [review]

**Comment by angadrandhawa1** [comment]
Reviewed the PR, it is ready to be merged.

**Review by angadrandhawa1 — APPROVED** [review]

**Comment by jpipkin1** [comment]
@kimsauce, it's been over an hour since this PR merged, but the release note is still not showing up at https://help.sumologic.com/release-notes-service. Do you have any idea why?

**Comment by jpipkin1** [comment]
Never mind, @kimsauce. The updates just showed up.

**Pull request merged**
